app/components/owned-card-list.cjsx

```diff
@@ -26,34 +27,45 @@ module.exports = React.createClass
   componentWillUnmount: ->
     document.documentElement.classList.remove 'on-secondary-page'
 
-  userForTitle: ->
-    if @props.ownerName
-      "#{@props.ownerName}'s"
-    else
-      'All'
+  getPageClasses: ->
+    classes = 'secondary-page all-resources-page'
+    if @props.project?
+      classes += ' has-project-context'
+    classes
```

Comment:
You could use [classnames](https://www.npmjs.com/package/classnames) for conditionally adding classes to components. It's already in `package.json` and a fairly clean and readable.